apps/demos/testing/widgets/contextmenu/ContextMenuShare.test.js

```diff
@@ -0,0 +1,25 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import { Selector as $ } from 'testcafe';
+import { runManualTest } from '../../../utils/visual-tests/matrix-test-helper';
+
+fixture('ContextMenu.Share')
+  .page('http://localhost:8080/')
+  .beforeEach(async (t) => {
```

Comment:
We use the following method to notify the fixture update window size for tests
```
.before(async (ctx) => {
    ctx.initialWindowSize = [900, 600];
  });
```
Your approach it will not work in CI, because each test that is running inside runManualTest has before() hook. Test before.hook() has more priority than fixture.beforeEach() hook according to testcafe documentation.

Reply:
You can find this approach in neighboring files.